lib/node_modules/@stdlib/types/index.d.ts

```diff
@@ -593,6 +593,85 @@ declare module '@stdlib/types/array' {
 	* const x: Collection<number> = [ 1, 2, 3 ];
 	*/
 	type Collection<T = any> = Array<T> | TypedArray | ArrayLike<T>; // eslint-disable-line @typescript-eslint/no-explicit-any
+
+	/**
+	* Type mapping for the following array data types:
+	*
+	* -   `float64`: double-precision floating-point numbers (IEEE 754)
+	* -   `float32`: single-precision floating-point numbers (IEEE 754)
+	*/
+	type RealFloatingPointArrayMap = { // eslint-disable-line @typescript-eslint/consistent-type-definitions
```

Comment:
My preference would be to follow the "DataType" naming convention, so `RealFloatingPointDataTypeMap`. Same applies below. This makes it easier to remember, at least for me, when writing imports when authoring declarations. Whereas previously I would have imported `RealFloatingPointDataType`, now I append `Map`.

Reply:
Renamed.